icon-loader.js

```diff
@@ -1,12 +1,20 @@
 const VIEWBOX_REGEX = /viewBox="([^"]*)"/;
 const SVG_REGEX = /(<svg[^>]*>|<\/svg>)/g;
 const FILL_REGEX = /fill="[^"]*"/g;
+// Issue with ESLint recognizing this as needing an object destructure
+// eslint-disable-next-line prefer-destructuring
+const MIN_FILL_LENGTH = 'fill="#000"'.length;
+
+function fillIsWhite(fill) {
+  const whiteFillRegex = fill.length > MIN_FILL_LENGTH ? /#ffffff/gi : /#fff/gi;
+  return fill.match(whiteFillRegex);
+}
 
 module.exports = function shopifyIconLoader(source) {
   this.cacheable(true);
 
   const finalSource = source.replace(FILL_REGEX, (fill) => {
-    return fill.includes('#FFF') ? 'fill="currentColor"' : '';
+    return fillIsWhite(fill) ? 'fill="currentColor"' : '';
```

Comment:
You can probably do this as `/['"]#fff(?:fff)?['"]/.test(fill)`

Reply:
way better regex!

just one change - i needed to use `fill.match(regex)` instead of the `.test()` you recommended.

using `.test()` would fail the `multiple-fills` tests _(catches the first instance of white and removes the second)_